src/extensions/default/bramble/main.js

```diff
@@ -22,6 +22,8 @@ define(function (require, exports, module) {
         UrlParams            = brackets.getModule("utils/UrlParams").UrlParams,
         Editor               = brackets.getModule("editor/Editor").Editor,
         Rewriter             = brackets.getModule("filesystem/impls/filer/lib/HTMLRewriter"),
+        ThemeManager         = brackets.getModule("view/ThemeManager"),
+        ThemePreferences     = PreferencesManager.getExtensionPrefs("themes"),
```

Comment:
Make a new file in `lib/Theme.js` and require that in here.  Then, move all your theme pref flipping, etc into that.  This file is already way too big and messy.